src/components/FilterPanel/filter-panel.js

```diff
@@ -1,11 +1,17 @@
 import React from "react";
 import FilterTimeline from "../FilterTimeline/filter-timeline";
-import FilterPanelButtons from "./filter-panel-buttons";
 import FilterSelection from "../FilterSelection/filter-selection";
 
 const FilterPanel = props => (
```

Comment:
there's some weird spacing: 
<img width="551" alt="Screenshot 2020-01-31 at 12 55 41" src="https://user-images.githubusercontent.com/13707737/73538579-aaea8480-442b-11ea-8dc6-5571fb347a53.png">
first of all on top (towards the buttons) the margins are gone, looks not on purpose

but also between the filters (should they really be evenly spaced) ? it seems more reasonable to top-align them and have some empty space towards the bottom, if necessary (design thing, I just think it looks weird)

Reply:
the spacing is what they wanted, the top margin is another thing i didnt commit